src/pages/api/public/health.ts

```diff
@@ -0,0 +1,10 @@
+import { cors, runMiddleware } from "@/src/features/publicApi/server/cors";
+import { type NextApiRequest, type NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  await runMiddleware(req, res, cors);
+  return res.status(200).json({ status: "OK" });
```

Comment:
huh, health endpunkt macht sinn, aber wie kommt der hier rein? Normalerweise würde man hier auch die infrastruktur testen. In unserem fall die db.

Reply:
wanted to add v1 of it, we can add more health checks to it in next releases